### PR TITLE
Es lint fixing 

### PR DESCRIPTION
**Pull request by KosSid**
1. es lint error message was fixed.
!!! client .env file was removed from gitignore to fix issue with eslint settings=> new variable was added in env => SKIP_PREFLIGHT_CHECK=true
2. husky & lint-staged were added.
!!! Note you will not be able to commit any changes in git in case you'll see active eslint errors or warnings in the terminal. To 
3. commit changes you must to fix errors.
Now you can launch project from both folders (root and client) with command $ npm run start

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +30,7 @@
    - work with pull requests => https://dan-it.gitlab.io/fe-book/final-project/pull_request.html  
    - Travis CI => https://dan-it.gitlab.io/fe-book/final-project/travis_ci.html 
     
-2. Note! master branch is secured. You can't push updates directly in the master. 
+2. Note! master and develop branches are secured. You can't push updates directly in the master. 
    We use development branch as a "middlaware" before merge with master.
    All pull requests reqire 2 Code reviews from your team members. See details in links above.
```
Comment (Anton-Sypavka) [inline README.md:35]: All pull requests require 6 Code reviews from team members.

**Review by Anton-Sypavka — COMMENTED** [review]
Approve in case of changes in Readme.md

**Review by alex-patsalov — APPROVED** [review]
ок

**Review by peksymov — APPROVED** [review]

**Comment by vrindislam** [comment]
approve

**Review by vrindislam — APPROVED** [review]
approve

**Review by Anton-Sypavka — APPROVED** [review]

File: client/.env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_API='http://localhost:5000/api'
+SKIP_PREFLIGHT_CHECK=true
```
Comment (alex-patsalov) [inline client/.env:2]: ?

File: client/.gitignore [hunk 3]
```diff
@@ -13,11 +13,12 @@
 
 # misc
 .DS_Store
-.env
```
Comment (alex-patsalov) [inline client/.gitignore:16]: ?

File: client/10 [hunk 4]
```diff
@@ -0,0 +1,8 @@
++ lint-staged@10.5.3
+added 41 packages from 34 contributors and audited 2502 packages in 17.612s
+
+157 packages are looking for funding
+  run `npm fund` for details
+
+found 2 vulnerabilities (1 low, 1 high)
+  run `npm audit fix` to fix them, or `npm audit` for details
```
Comment (alex-patsalov) [inline client/10:8]: зачем?

**Pull request closed without merging**